src/ngAria/aria.js

```diff
@@ -83,7 +83,8 @@ function $AriaProvider() {
     ariaMultiline: true,
     ariaValue: true,
     tabindex: true,
-    bindKeypress: true
+    bindKeypress: true,
+    keypressNodeNames: ['DIV', 'LI']
```

Comment:
Fair point. `<button>` is the biggest issue we had, since it gets both click and keypress. `<a>` should not be used with `ng-click` (that's what buttons are for) but rather `ng-href` and routing...although I suppose that could be abused. There's also the `<details>`/`<summary>` elements, `<dialog>`, form controls, and [a few more](http://www.w3.org/html/wg/drafts/html/master/interactive-elements.html). An interesting thing to consider, as well, is the work being done on Custom Elements and tabindex: https://docs.google.com/document/d/1k93Ez6yNSyWQDtGjdJJqTBPmljk9l2WS3JTe5OHHB50/edit#heading=h.tf9twftfr65h

So, we need some mechanism of checking whether or not to apply keypress–I'm personally in favor of opt-in for `<div>` and `<li>` so that it limits the scope of this behavior. It's just a question of whether we want that list to be configurable.